### PR TITLE
adds usage information and example

### PR DESCRIPTION
**Pull request by gabrielcsapo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,25 @@
...
 ```
 
+## Usage
+
+```js
+const Session = require('@allegiant/sessions');
+const App = require('@allegiant/core');
+
+let server = App.create('http://localhost:7000');
+Session.configure(server); // will by default configure needed sesion defaults
```
Comment (echopoint) [inline README.md]: session is mispelled as 'sesion'

Comment (echopoint) [reply README.md]: Also, there isn't a need to do Session.configure directly if you want to store it in the app configuration.. the app will load and bind the module at the right time in this setup:
```js
var server = App.create("https://localhost:7000", { 
    '@allegiant/sessions': {
        enabled: true,
        path: path.resolve(path.join(process.cwd(), 'sessions')),
    }
});
```

All the modules for the fw work this way, which is what configure is for.

**Comment by gabrielcsapo** [comment]
@echopoint updated!

**Comment by echopoint** [comment]
@gabrielcsapo If you don't see anything else, it's good to go

**Review by echopoint — APPROVED** [review]

**Comment by gabrielcsapo** [comment]
@echopoint awesome! looks good to me!

**Pull request merged**